docs/TVM_EP.md

```diff
@@ -42,26 +43,43 @@ sudo apt-get install kitware-archive-keyring
 sudo apt-get install cmake
 ```
 
-Build ONNX Runtime:
-```
-./build.sh --config Release --enable_pybind --build_wheel --skip_tests --parallel --use_stvm --skip_onnx_tests
+Build ONNX Runtime (TVM x86):
+```bash
+./build.sh --config Release --enable_pybind --build_wheel --parallel --skip_tests --skip_onnx_tests --use_tvm
 ```
 
-This command builds both TVM and onnxruntime-stvm. It creates two wheel, one for each project.
-Build the python API for ONNX Runtime instead of using the standard package:
+Build ONNX Runtime (TVM with CUDA support):
+```bash
+./build.sh --config Release --enable_pybind --build_wheel --parallel --skip_tests --skip_onnx_tests --use_tvm --tvm_cuda_runtime
 ```
+
+This command builds both `TVM` and `onnxruntime-tvm`. It creates two wheel, one for each project.
+Build the python API for ONNX Runtime instead of using the standard package. Instructions for this are given below.
+
+Package for TVM:
+```bash
 cd <path_to_onnx_runtime>
-pip3 uninstall onnxruntime onnxruntime-stvm tvm -y
-whl_path=$(find ./build/Linux/Release/dist -name "*.whl")
+python3 -m pip uninstall tvm -y
+whl_path=$(find ./build/<OS_NAME>/Release/_deps/tvm-src/python/dist -name "*.whl")
```

Comment:
Good catch. linux or os_name is not mandatory. It can be removed.

Reply:
We assume that ONNX runtime is built by build.sh (see TVM_EP.md). Inside the latter the path with OS_DIR is constructed due to this linux or os_name is mandatory